api/swagger.yaml

```diff
@@ -2976,10 +2980,14 @@ definitions:
         description: "A list of environment variables used during stack deployment"
         items:
           $ref: "#/definitions/Stack_Env"
+      Prune:
+        type: "boolean"
+        example: true
+        description: "Prune services that are no longer referenced?"
```

Comment:
Please remove the last `?` char here.

Reply:
I'll prepare the changes. Thanks a lot for your feedback.